conans/test/integration/toolchains/ios/test_using_cmake.py

```diff
@@ -0,0 +1,80 @@
+import platform
+import textwrap
+import unittest
+
+from conans.client.toolchain.cmake.base import CMakeToolchainBase
+from conans.test.utils.tools import TestClient
+from ._utils import create_library
+
+
+@unittest.skipUnless(platform.system() == "Darwin", "Requires XCode")
+class ToolchainiOSTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.t = TestClient()
+        create_library(self.t)
+        self._conanfile = textwrap.dedent("""
+            from conans import ConanFile, CMake, CMakeToolchain
+
+
+            class Library(ConanFile):
+                name = 'hello'
+                version = '1.0'
+                settings = 'os', 'arch', 'compiler', 'build_type'
+                exports_sources = 'hello.h', 'hello.cpp', 'cpp-wrapper.h', 'cpp-wrapper.mm', 'CMakeLists.txt'
+                options = {{'shared': [True, False]}}
+                default_options = {{'shared': False}}
+                _cmake = None
+
+                def _configure_cmake(self):
+                    if not self._cmake:
+                        self._cmake = CMake(self, generator={generator}, parallel=False)
+                        self._cmake.configure()
+                    return self._cmake
+
+                def toolchain(self):
+                    tc = CMakeToolchain(self)
+                    tc.write_toolchain_files()
+
+                def build(self):
+                    cmake = self._configure_cmake()
+                    cmake.configure()
+                    cmake.build()
+                    self.run("lipo -info Release-iphoneos/libhello.a")
```

Comment:
Is this very needed? Should be part of the toolchain somehow? Can't be done by cmake? Is expected that the user runs this command manually in the terminal when building without conan?

Reply:
That's to check later in the test that the library we have compiled actually resulted with `arm64` architecture.